src/main/java/qna/domain/Question.java

```diff
@@ -81,8 +89,19 @@ public boolean isDeleted() {
         return deleted;
     }
 
-    public void setDeleted(boolean deleted) {
-        this.deleted = deleted;
+    public List<DeleteHistory> delete(User loginUser) throws CannotDeleteException {
```

Comment:
```suggestion
    public List<DeleteHistory> delete(User loginUser) {
```

RuntimeException으로 변경하신 만큼 제거해볼 수 있겠네요~